@ndlib/gatsby-theme-marble/src/components/Internal/FeedbackModal/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import ReactModal from 'react-modal'
+import FeedbackForm from '../FeedbackForm'
+
+import style from './style.module.css'
+
+export class FeedbackModal extends Component {
```

Comment:
Again, we're trying to avoid class components, this should be handled with `useState`.